assets/src/scss/layout/_country-selector.scss

```diff
@@ -0,0 +1,195 @@
+
+#country-selector {
+  --country-selector-- {
+    color: $white;
+    font-size: $font-size-sm;
+  }
+  margin-bottom: 24px;
+  text-align: start;
+  width: 100%;
+
+  a --country-selector--a-- {
+    color: $white;
+  }
+
+  a:hover,
+  button:hover {
+    text-decoration: var(--link--hover--text-decoration, underline);
+  }
+
+  button:focus {
+    border-radius: var(--link--focus--border-radius, 4px);
+    outline: var(--link--focus--border, 2px solid rgba(255, 255, 255, 0.4));
+    outline-offset: 0;
+  }
+
+  button:focus:not(:focus-visible) {
+    outline: 0;
+  }
+
+  .country-selector-toggle-container {
+    --country-selector--toggle-container-- {
+      border-bottom: 1px solid rgba(255, 255, 255, 0.5);
+    }
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding-bottom: 24px;
+    transition: border 0s linear 0.5s;
+    vertical-align: middle;
+    width: 100%;
+  }
+
+  .control-toggle {
+    align-self: flex-start;
+    background: transparent;
+    border: none;
+    color: var(--site-footer--color, $white);
+    cursor: pointer;
+    text-align: start;
+    white-space: nowrap;
+    width: 100%;
+
+    &::before {
+      background-color: var(--site-footer--color, $white);
+      content: "";
+      display: inline-block;
+      height: 20px;
+      margin-inline-end: 8px;
+      mask: url("../../images/gp-g-logo.png") 0 0/16px 20px no-repeat;
+      vertical-align: middle;
+      width: 16px;
+    }
+
+    &::after {
+      background-color: var(--site-footer--color, $white);
+      content: "";
+      display: inline-block;
+      height: 12px;
+      margin-inline-start: 8px;
+      mask: url("../../images/chevron.svg") 0 0/16px 12px no-repeat;
+      transform: rotate(0.25turn);
+      transition: transform 300ms cubic-bezier(.86, 0, .07, 1);
+      width: 16px;
+    }
+  }
+
+  .countries-list {
+    --country-selector--list-- {
+      background-color: $active-blue;
+      font-weight: 500;
+    }
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height 0.5s cubic-bezier(0, 1.04, .38, .37) 0s;
+  }
+
+  .countries-list .container {
+    padding: 24px 24px $space-lg 16px;
+
+    @include medium-and-up {
+      padding: 24px 0 $space-lg 0;
+    }
+  }
+
+  &.open {
+    .country-selector-toggle-container {
+      border-bottom: 1px solid rgba(255, 255, 255, 0);
+      transition-delay: 0s;
+    }
+
+    .control-toggle::after {
+      transform: rotate(-0.25turn);
+    }
+
+    .countries-list {
+      max-height: 5000px;
+      transition: max-height 1s linear 0s;
+    }
+  }
+
+  ul.countries {
+    column-count: 1;
```

Comment:
`column-count` has a drawback we discovered on the footer, which probably is hard to get around. Not sure if it leads to other similar issues.

Check how the focus outline is split over 2 columns.

![Screenshot from 2021-11-16 13-38-46](https://user-images.githubusercontent.com/7604138/141986846-f5395594-5dca-4613-85ce-66c16c3c8fbb.png)

Reply:
Seems limited to Chrome (doesn't happen on Firefox or Safari for me) ? I checked in their bugs, on a semi-related bug they [mention](https://bugs.chromium.org/p/chromium/issues/detail?id=269061#c31) working on a new thing that would fix rendering issues with columns, but not sure it would include this one.

The country selector also has this crop from the top list element, but it doesn't appear on the bottom of the previous column.
Adding a sufficient `padding-top` to the `li` element would work, that probably also means changing other sizes. Considering the low impact I would prefer to open a bug ticket for this and look for a proper solution later.

Reply:
Interesting, I think last time I checked Firefox for this and it also had the issue, but now indeed it doesn't. Perhaps they did a fix.

Reply:
Definitely not a blocker for me, just highlighting it so we can check whether an alternative makes sense.